fw/apm.c

```diff
@@ -1953,6 +2107,13 @@ static TfwCfgSpec tfw_apm_specs[] = {
 		.allow_repeat	= true,
 		.cleanup	= tfw_cfgop_apm_cleanup_server_failover,
 	},
+	{
+		.name		= "health_stat_server",
+		.deflt		= NULL,
+		.handler	= tfw_cfgop_apm_health_stat_srv,
+		.allow_none	= true,
+		.cleanup	= tfw_cfgop_apm_cleanup_server_failover,
```

Comment:
Is this correct or is this a copy-paste error? Using the same cleanup function with such a specific name is confusing.